orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/OrchSessionService.java

```diff
@@ -158,6 +158,10 @@ public Optional<OrchSessionItem> getSessionFromRequestHeaders(Map<String, String
         return sanitiseBase64(sessionId).flatMap(id -> getSession(sessionId));
     }
 
+    public void deleteStoredOrchSession(String sessionId) {
+        delete(sessionId);
```

Comment:
Is there any value wrapping this in a try / catch like we do with the other calls to dynamo?